### ISSUE TITLE
Warning after exporting repeat_interleave to onnx with dynamic axes

### ISSUE THREAD

**Issue by MartynaKsyta** [microsoft/onnxruntime#10783]
### 🐛  Bug 
Hi! I export to onnx a model with repeat_interleave and dynamic_axes. When I try to run it in onnxruntime session I get a warning and some tensors are printed:

> [10, 20] [10, 20]
> 2022-02-18 09:05:52.973973100 [W:onnxruntime:, constant_folding.cc:202 ApplyImpl] Unsupported output type of  
> N11onnxruntime22SequenceTensorTypeBaseE. Can't constant fold SequenceEmpty node 'SequenceEmpty_18'
> 2022-02-18 09:05:52.974430500 [W:onnxruntime:, constant_folding.cc:202 ApplyImpl] Unsupported output type of  
> N11onnxruntime22SequenceTensorTypeBaseE. Can't constant fold SequenceEmpty node 'SequenceEmpty_18'
> 2022-02-18 09:05:52.974665000 [W:onnxruntime:, constant_folding.cc:202 ApplyImpl] Unsupported output type of 
> N11onnxruntime22SequenceTensorTypeBaseE. Can't constant fold SequenceEmpty node 'SequenceEmpty_18'

This started to happening after I updated pytorch to the latest version.

### Urgency
Model gives correct results, so it's not urgent. 

### System information
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Debian GNU/Linux 11 (bullseye)
- ONNX Runtime installed from (source or binary): using pip (onnxruntime==1.10.0)
- ONNX Runtime version: 1.10.0
- Python version: 3.7.12 (default, Feb  8 2022, 05:37:08)  [GCC 10.2.1 20210110] (64-bit runtime)
- Visual Studio version (if applicable):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:  N/A
- GPU model and memory:  N/A
- Pytorch version:  1.10.2+cu102

### To Reproduce

Below is a minimal example:

```
import numpy as np
import onnxruntime
import torch


class MyModel(torch.nn.Module):
    def forward(self, input_tensor: torch.Tensor, repeats: torch.Tensor) -> torch.Tensor:
        return torch.repeat_interleave(input_tensor, repeats, dim=0)


if __name__ == "__main__":
    model = MyModel().eval()
    data = torch.randn(10, 20)
    repeats = torch.tensor(np.arange(1, 11))
    model_path = "/tmp/model.onnx"
    torch.onnx.export(
        model,
        (data, repeats),
        model_path,
        export_params=True,
        opset_version=13,
        do_constant_folding=False,
        input_names=["input_tensor", "repeats"],
        dynamic_axes={"repeats": {0: "name"}}
    )

    ort_session = onnxruntime.InferenceSession(model_path)
    ort_inputs = {"input_tensor": data.numpy(), "repeats": np.arange(1, 11)}
    ort_session.run(None, ort_inputs)

```

### Expected behavior 
Neither should warnings be displayed nor tensors printed.

**Comment by garymm** [comment]
Doesn't appear to be a converter bug since the model is exported successfully and the outputs between ORT and PyTorch match. Removing converter-pytorch label.

**Comment by yuslepukhin** [comment]
This means a constant folding optimization cannot be applied to certain constants because of the unsupported data type. This causes the model to run somewhat slower than it would otherwise. What are the data types for the nodes mentioned?
